src/dragonball/src/dbs_pci/src/lib.rs

```diff
@@ -30,22 +30,34 @@ mod bus;
 mod configuration;
 mod device;
 
+pub use self::bus::PciBus;
```

Comment:
Could we change the style like below？

mod xxx;
pub use xxx:yyy;

Reply:
modified.